lib/sass/calcite-web/grid/_columns.scss

```diff
@@ -234,4 +234,10 @@
   }
 
   @include responsive-column();
+
+  .center-column {
+    margin-left: auto !important;
```

Comment:
Why is `!important` necessary? Could it include a note?

In general, I think there should always be an explanation for `!important`.